apps/enc.c

```diff
@@ -243,6 +246,11 @@ int enc_main(int argc, char **argv)
                 goto opthelp;
             cipher = c;
             break;
+        case OPT_ITER:
```

Comment:
A simple google query ("pbkdf iteration count best practice") gives some answers:

http://security.stackexchange.com/questions/3959/recommended-of-iterations-when-using-pkbdf2-sha256